components/common/posts/PostsFeed.tsx

```diff
@@ -233,12 +233,14 @@ export function PostEmbeds({ item }: { item: AppBskyFeedDefs.FeedViewPost }) {
   )
 
   if (AppBskyEmbedVideo.isView(embed)) {
+    const captions = item.post.record?.['embed']?.['captions']
```

Comment:
Where the captions live in the record depends on whether it's a video embed, or a record-with-media embed.  Would recommend using the lexicon type guards to figure out where to pull it from (will also make it less likely that you get an invalid `captions` value that causes a UI crash).

Reply:
ok I'm gonna merge this for now since this works. for cases that I'm missing here, will circle back and plug those up tomorrow.